tests/integration/operator/uninstall_test.go

```diff
@@ -171,7 +171,7 @@ func TestReconcileDelete(t *testing.T) {
 					}
 
 					return nil
-				}, retry.Timeout(deletionTimeout), retry.Delay(retryDelay)); err == nil {
+				}, retry.Timeout(deletionTimeout), retry.Delay(retryDelay), retry.MaxAttempts(10)); err == nil {
```

Comment:
fix test timeout failed.

I have some doubts about this test case. My understanding is that the ingressgateway should not impede the uninstallation of the operator (IOP).
https://github.com/istio/istio/blob/2a1a725154f0d6552e6f8e85976fd59c7fe80ee1/tests/integration/operator/uninstall_test.go#L113-L178